crates/node/builder/src/launch/engine.rs

```diff
@@ -65,36 +67,42 @@ impl EngineNodeLauncher {
     }
 }
 
-impl<Types, DB, T, CB, AO> LaunchNode<NodeBuilderWithComponents<T, CB, AO>> for EngineNodeLauncher
+impl<T> LaunchNode<NodeBuilderWithComponents<T>>
+    for EngineNodeLauncher
 where
```

Comment:
hmm, I see because we removed the adapter type here this requires more bloat now?